2022-07_sprint/sprint_parties/community_partners.md

```diff
@@ -1,4 +1,29 @@
-# Community Partners
+# Community Partnership
+
+## Become a Community Partner
+
+Community Partners are organizations who support our mission and share our event for outreach.  Data Umbrella will add the Name, Logo and Website Link to this page to bring visibility to your community.  If you would like to be a community partner, please contact us (info@dataumbrella.org).
+
+Our event series can be shared in one or more of the following ways:  
+- Share the event website:   https://pymc-data-umbrella.xyz/en/latest/2022-07_sprint/schedule.html
+- Share the event registration form:  https://tinyurl.com/du-pymc-os-ws
+- Tweet:  [to be added]
+- LinkedIn post (English): [to be added]
+- Social media card:  https://drive.google.com/file/d/18jwsvoWTxgj-Ex3AQWdK_AtPbgJm0Bm9/view?usp=sharing
+- on Slack, Discord, Facebook
+- Email copy
+>We [insert org name] are excited to share that we are a Community Partner with Data Umbrella for their upcoming open source working sessions with the Python library, PyMC.  PyMC is an open-source probabilistic programming language in Python.
+
+>This is a beginner-friendly event.  Skills that are helpful to have: Git, GitHub, Markdown, Sphinx, statistics, python, Bayesian statistics.
+
+>Additionally, we have a series of videos where participants can watch videos and add timestamps. There are lots of ways to participate and contribute to open source.
+
+>More information on joining the event:   
+a) Schedule of events: https://pymc-data-umbrella.xyz/en/latest/2022-07_sprint/schedule.html  
+b) Complete this registration form: https://tinyurl.com/du-pymc-os-ws
```

Comment:
````suggestion
```
We [insert org name] are excited to share that we are a Community Partner with Data Umbrella for their upcoming open source working sessions with the Python library, PyMC.  PyMC is an open-source probabilistic programming language in Python.

This is a beginner-friendly event.  Skills that are helpful to have: Git, GitHub, Markdown, Sphinx, statistics, python, Bayesian statistics.

Additionally, we have a series of videos where participants can watch videos and add timestamps. There are lots of ways to participate and contribute to open source.

More information on joining the event:   
a) Schedule of events: https://pymc-data-umbrella.xyz/en/latest/2022-07_sprint/schedule.html  
b) Complete this registration form: https://tinyurl.com/du-pymc-os-ws
```
````

To render things as quotes in markdown there needs to be a space between the `>` and the text, it should also be on all lines, empty lines break the quote and start a different one. Doing that looks like

> quoted text here

However, here I think ease of copying is more important than how it renders, so we can use a code block so it will have the copy button at the top right and it

Reply:
@OriolAbril Yes, here ease of copying is key, so people can copy the info into emails, etc.